hbase-server/src/main/java/org/apache/hadoop/hbase/regionserver/HRegion.java

```diff
@@ -4690,9 +4710,10 @@ private OperationStatus[] batchMutate(BatchOperation<?> batchOp) throws IOExcept
         requestFlushIfNeeded();
       }
     } finally {
-      if (rsServices != null && rsServices.getMetrics() != null) {
```

Comment:
can we revert this diff back to `if (this.rsServices != null && this.rsServices.getMetrics() != null)`? same below